### ISSUE TITLE
Chrome unter Ubuntu zeigt kein &#x0364;

### ISSUE THREAD

**Issue by mhug** [deutschestextarchiv/dtabf#73]
Bsp.: http://www.deutschestextarchiv.de/doku/basisformat/geLyrikband.html

Gibt es einen Hinweis über bevorzugte Browser (Firefox fkt.) den ich übersehen habe?

**Comment by funkyfuture** [comment]
wenn die frage lautet, ob zum lesen der DTABf-doku bestimmte technische voraussetzungen erfüllt sein müssen: die verwendeten fonts müssen die fraglichen codepoints unterstützen. Chromium unter Ubuntu Disco zeigt bspw. `oͤ` (`o` mit hochgestelltem `e` (U+0364)) korrekt an. afaict ist dafür nicht das installieren eines bestimmten fonts notwendig.

**Comment by mhug** [comment]
Mmh, mysteriös. Bei mir (ubuntu bionic) wird das hochgestellte e im Browser (betrifft Google Chrome und Chromium) nicht angezeigt, sondern ersetzt durch ein kleines Quadrat.

**Comment by StefanDumont** [comment]
Korrekte Anzeige von Schrift hängt vom verwendeten Font, vom Browser und auch vom Betriebssystem (sowie teilweise dessen Konfiguration) ab. Das ist sehr aufwändig da was dran zu drehen, insbesondere wenn es um einzelne Buchstaben geht.

**Comment by cthomasdta** [comment]
> Korrekte Anzeige von Schrift hängt vom verwendeten Font, vom Browser und auch vom Betriebssystem (sowie teilweise dessen Konfiguration) ab. Das ist sehr aufwändig da was dran zu drehen, insbesondere wenn es um einzelne Buchstaben geht.

Demnach kann das issue hier zu, betrifft ja nicht DTABf-Entwicklung?

**Comment by funkyfuture** [comment]
Doch, zumindest wenn eine breite Rezeption des Entwicklungsergebnisses zu den Entwicklungszielen zählen sollte.

**Comment by cthomasdta** [comment]
@mhug ist das hier noch aktuell? Wenn nicht, bitte schließen. Danke!
